services/workflows-service/prisma/migrations/20230921074435_enable_is_legacy_reject_flag_for_existing_workflow_definitions/migration.sql

```diff
@@ -0,0 +1 @@
+UPDATE "WorkflowDefinition" SET "config" = jsonb_set(COALESCE("config", '{}'::jsonb), '{isLegacyReject}', 'true'::jsonb);
```

Comment:
I dont think it should be in the migrations, we can't be sure in the future it will always run on data expect it to run on - it should be a one time thing done on a specific environment with a standalone script like the migration script